### PR TITLE
Icon typo fixes (used non transparent gif by accident)

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-XXXXX](https://issues.jenkins-ci.org/browse/JENKINS-XXXXX).

Regression in https://github.com/jenkinsci/jenkins/pull/5065

When I was asked to revert the sidepanel changes I made a couple of typoes and accidentally used the non transparent gifs for a few icons, which looks really bad in dark theme.

### Proposed changelog entries

* Entry 1: Gear clipboard, and user icons are transparent again, regression in 2.283
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by oleg-nenashev — APPROVED** [review]

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (daniel-beck) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:508]: What's with line 532?

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (daniel-beck) [inline core/src/main/java/org/jenkins/ui/icon/IconSet.java:609]: line 570?

Comment (timja) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:609]: I don't know why gif and png are in here, historical I assume, when felix switched things to svgs he removed the duplicates, we need to do the same cleanup when we switch the rest to SVG

File: core/src/main/java/org/jenkins/ui/icon/IconSet.java
Comment (timja) [reply core/src/main/java/org/jenkins/ui/icon/IconSet.java:508]: I don't know why gif and png are in here, historical I assume, when felix switched things to svgs he removed the duplicates, we need to do the same cleanup when we switch the rest to SVG

https://github.com/jenkinsci/jenkins/pull/5065/files#r587464783

**Review by fqueiruga — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by res0nance — APPROVED** [review]

**Pull request merged**
